app/services/gpo_reminder_sender.rb

```diff
@@ -7,6 +7,7 @@ def send_emails(for_letters_sent_before)
     reminder_eligible_range =
       IdentityConfig.store.usps_confirmation_max_days.days.ago..for_letters_sent_before
     profiles_due_for_reminder(for_letters_sent_before).each do |profile|
+      next if profile.user.active_profile
```

Comment:
I spent a few minutes trying to adjust the ActiveRecord query to just find profiles for users that don't have an active profile, but gave up and went with this. The numbers are pretty small in production, so there's not too much associated waste with doing this filtering in Ruby code rather than in SQL.